test/.eslintrc.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+  env: {
+    mocha: true,
+    node: true
+  },
+  rules: {
+    'no-unused-expressions': 0,
+    'arrow-body-style': 0
+  },
+  globals: {}
+}
```

Comment:
I think this is a good addition :+1:. 

Though, this should be in another PR, to test it and release it separately. It could have been reviewed and accepted quickly in another PR (imagine if this one is not accepted, the eslint rules would not be accepted too, even if it's a good addition).

Reply:
(or maybe I misunderstood something, but still, there could have been a comment on the PR)

Reply:
As you requested: https://github.com/ForestAdmin/forest-express-sequelize/pull/332

Reply:
I'll rebase once validated